scripts/install.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+{ # this ensures the entire script is downloaded #
+
+BINARY_DEST=${2:-/usr/local/bin}
+BINARY_NAME="rhoas"
+SRC_ORG="bf3fc6c"
+SRC_REPO="cli"
+OS_TYPE="linux"
+OS_LONG_BIT=""
+API_BASE_URL="https://api.github.com"
+API_RELEASE_URL="${API_BASE_URL}/repos/${SRC_ORG}/${SRC_REPO}/releases/latest"
+DOWNLOAD_DIR="/tmp"
+
+has_in_path() {
+  type "$1" > /dev/null 2>&1
+}
+
+source_file() {
+  # shellcheck disable=SC1090
+  source "$1" 2> /dev/null
+}
+
+# get OS type
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  OS_TYPE="macOS"
+fi
+
+MACHINE_TYPE=$(uname -m)
+if [ "${MACHINE_TYPE}" == 'x86_64' ]; then
+  OS_LONG_BIT="64"
+else
+  OS_LONG_BIT="386"
```

Comment:
My take would be to not have that. 

All red hat cli keep things simple to 3 binaries:

linux mac windows (all 64)

https://github.com/openshift/origin/releases